bin/expire-sessions

```diff
@@ -19,41 +19,37 @@ use Getopt::Long;
 use List::Util qw(uniq);
 
 GetOptions(
+    # Update sessions to make sure all present in User objects
     'init' => \my $init,
 );
 
 my $rs = FixMyStreet::DB->resultset("Session");
+my $now = time();
 
 # Delete expired sessions (including from in User object)
-while (my $session = $rs->search({ expires => { '<', time() } })->next) {
```

Comment:
Could we not leave this search in if init is missing to avoid looping over every session in the DB? I guess it's probably not a big issue given this is happening offline.

Reply:
It has to loop over all sessions to set the last active time, not just when init is set (the `if $user` bit always runs).